tensorflow/core/graph/mkl_tfconversion_pass.cc

```diff
@@ -123,22 +124,22 @@ Status MklToTfConversionPass::InsertConversionNodeOnEdge(
   TF_CHECK_OK(GetNodeAttr(src->def(), "T", &src_datatype));
   TF_CHECK_OK(GetNodeAttr(dst->def(), "T", &dst_datatype));
   if (src_datatype != dst_datatype) {
-    string err_msg = "T attribute of " + src->name() + " and " + dst->name() +
-                     " do not match. Will not insert" +
+    string err_msg = "T attribute of " + src->name() + " and " +
```

Comment:
Line 94: "edge will be deleted ...", could you point to me in the code where the deletion of edge is?